packages/deployer/test/fixture-projects/complete-run/contracts/Proxy.sol

```diff
@@ -0,0 +1,30 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./storage/ProxyNamespace.sol";
+
+contract Proxy is ProxyNamespace {
```

Comment:
Ass discussed in call, let's not duplicate this code and try to re-use sample-project.

It's ok to lose sample-project deployment data.

Reply:
done!